lib/cli/watch-run.js

```diff
@@ -167,8 +204,10 @@ const createWatcher = (
   // we handle global fixtures manually
   mocha.enableGlobalSetup(false).enableGlobalTeardown(false);
 
-  const watcher = chokidar.watch(watchFiles, {
-    ignored: watchIgnore,
+  const tracker = new GlobFilesTracker(watchFiles, watchIgnore);
+  tracker.regenerate();
+
+  const watcher = chokidar.watch('.', {
```

Comment:
@ggodlewski if you're available, I'm debugging #5355 and curious why we changed this to `watch('.'` from `watch(watchFiles`

Reply:
You're right, thanks :D the author of the mentioned incident linked the same stuff. Thanks for your patience, I'm still getting my bearings here :)